tests/php/src/ExtraThemeAndPluginHeadersTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace AmpProject\AmpWP\Tests;
+
+use AmpProject\AmpWP\Infrastructure\Registerable;
+use AmpProject\AmpWP\Infrastructure\Service;
+use AmpProject\AmpWP\ExtraThemeAndPluginHeaders;
+use WP_UnitTestCase;
+
+/** @coversDefaultClass \AmpProject\AmpWP\ExtraThemeAndPluginHeaders */
+final class ExtraThemeAndPluginHeadersTest extends WP_UnitTestCase {
+
+	/** @var ExtraThemeAndPluginHeaders */
+	private $instance;
+
+	public function setUp() {
+		parent::setUp();
+		$this->instance = new ExtraThemeAndPluginHeaders();
+	}
+
+	/** @covers ::__construct() */
```

Comment:
I noticed this will cause a warning when the test is run with coverage, since the method technically is not present in the class.

```suggestion
```